en/3-meeds.md

```diff
@@ -16,15 +16,15 @@ The rest is part of the __Meeds DAO funds__ and will be used to pay for services
 
 Minted Meeds are allocated to encourage involvement of the three categories of stakeholders - the users, the builders and the investors:
 
-- The users (users and Deeds holders/renters) get Meeds through engagement rewards (mintium model).
+- The users (users and Deed holders/renters) get Meeds through engagement rewards (mintium model).
 - Builders (members of the association and community) get Meeds through the association funds to build, operate and improve the WoM.
 - Investors mint Meeds through renting and buying of liquidity pools and staking Meeds.
 
 At the beginning, the builders get 30% of the allocation. The 70% are shared between the users and the investors. This allocation proportion will be reviewed on a regular basis, through the Meeds association DAO vote.
 
 ## 3.3. Users – minting through engagement
 
-Part of the continuously minted Meeds is allocated to the engagement minting channel. The distribution is done weekly. At any given week end, the engagement index is calculated and Deeds can claim their rewards the following week and get their Meeds. Unclaimed Meeds stay in the fund and are added to the next week’s distribution.
+Part of the continuously minted Meeds is allocated to the engagement minting channel. The distribution is done weekly. At any given week end, the engagement index is calculated and Hubs and Deed owners can claim their rewards the following week and get their Meeds. Unclaimed Meeds stay in the fund and are added to the next week’s distribution.
```

Comment:
thanks for the question @Bigemaa, I will consider clarification on a separate task because this one is focused on Disambiguating the Deed term. could you open a new issue for it ?
👉 https://github.com/Meeds-io/whitepaper/issues